rt_dashboard/javascript/src/components/dashboard/dashboard.js

```diff
@@ -20,7 +20,9 @@ export default class Dashboard extends HTMLElement {
   }
 
   selectedQueueChange (event) {
-    const { detail: { queueName } } = event
-    console.log(queueName)
+    const { detail: { queueName, count } } = event
+    const tasksComponent = this.shadowRoot.querySelector('tasks-component')
+    tasksComponent.setAttribute('queue', queueName)
+    tasksComponent.setAttribute('count', count)
```

Comment:
could use props, at least for the count that would not convert the number to string